Meta.Tests.ps1

```diff
@@ -1,57 +1,52 @@
 <# 
-.summary
-    Test that describes code.
-#>
+    .summary
+        Test that describes code.
 
+    .PARAMETER Force
+        Used to force any installations to occur without confirming with
+        the user.
+#>
 [CmdletBinding()]
-param()
+Param (
+    [Boolean]$Force = $false
+)
 
 if (!$PSScriptRoot) # $PSScriptRoot is not defined in 2.0
 {
     $PSScriptRoot = [System.IO.Path]::GetDirectoryName($MyInvocation.MyCommand.Path)
 }
+# Make sure MetaFixers.psm1 is loaded - it contains Get-TextFilesList
+Import-Module -Name (Join-Path -Path $PSScriptRoot -ChildPath 'MetaFixers.psm1') -Force
+
+# Load the TestHelper module which contains the *-ResourceDesigner functions
+Import-Module -Name (Join-Path -Path $PSScriptRoot -ChildPath 'TestHelper.psm1') -Force
 
 $ErrorActionPreference = 'stop'
 Set-StrictMode -Version latest
 
 $RepoRoot = (Resolve-Path $PSScriptRoot\..).Path
 
-Import-Module $PSScriptRoot\MetaFixers.psm1
-
-#region Download and import latest stable DSC Resource Designer module
-
-# We are using nuget to be compatible with WMF v4 machines which
-# do not have the PowerShell Package Management module.
-$DesignerModuleName = 'xDscResourceDesigner'
-$DesignerModulePath = "$env:USERPROFILE\Documents\WindowsPowerShell\Modules\$DesignerModuleName"
-$nugetSource = 'https://www.powershellgallery.com/api/v2'
-$nugetOutputDirectory = "$(Split-Path -Path $DesignerModulePath -Parent)\\"
-
-
-if (Test-Path -Path $DesignerModulePath)
-{
-    # Remove any installed version of the DscResourceDesigner module
-    Remove-Item -Path $DesignerModulePath -Recurse -Force
+# Install and/or Import xDSCResourceDesigner Module
+if ($env:APPVEYOR) {
+    # Running in AppVeyor so force silent install of xDSCResourceDesigner
+    $PSBoundParameters.Force = $true
```

Comment:
Looks like because of this the workaround which you mentioned in the conversation is no longer necessary, right?

```
2. Alternately, because this code has been moved into the TestHelper.psm1, the following just needs to be fired once before calling Invoke-Pester:
Import-Module TestHelper.psm1
Install-ResourceDesigner -force


So this will allow both CI scenarios (non-appveyor) and user interactive scenarios, so the user won't be just thrown a bunch of errors and give up.

This will install the module silently (no confirmation) if it is not installed and then import it. If it is already installed it'll just import it.

```